orc8r/tools/packer/magma-test-virtualbox.json

```diff
@@ -82,10 +81,7 @@
       "expect_disconnect": true
     },
     {
-      "extra_arguments": [
```

Comment:
Is this change done with a formatter?

Reply:
All VM `.json`s are now formatted the same. The order is debatable and was also topic during the last PR releasing base images.